core/domain/question_domain.py

```diff
@@ -265,6 +265,30 @@ def _convert_state_v31_dict_to_v32_dict(cls, question_state_dict):
 
         return question_state_dict
 
+    @classmethod
+    def _convert_state_v32_dict_to_v33_dict(cls, question_state_dict):
+        """Converts from version 32 to 33. Version 33 adds a new
+        customization arg to MultipleChoiceInput which allows
+        answer choices to be shuffled.
+
+        Args:
+            question_state_dict: dict. A dict where each key-value pair
+                represents respectively, a state name and a dict used to
+                initialize a State domain object.
+
+        Returns:
+            dict. The converted question_state_dict.
+        """
+        if question_state_dict['interaction']['id'] == 'MultipleChoiceInput':
+            customization_args = question_state_dict[
+                'interaction']['customization_args']
+            customization_args.update({
+                'showChoicesInShuffledOrder': {
+                    'value': True
```

Comment:
Not so much an issue here as there aren't a lot of questions in the server.

Reply:
Yes, I agree this is probably OK.